cpp/tests/test_time_series.cpp

```diff
@@ -342,20 +343,138 @@ TYPED_TEST(TestTimeSeries, print_table)
 
     output.str("");
 
-    std::string expected_output_2 = "\nTime  #1     #2    \n   0.0    0.1    1.1\n   1.0    1.1    2.1\n";
+    std::string expected_output_2 = "\nTime   #1     #2    \n   0.0    0.1    1.1\n   1.0    1.1    2.1\n";
```

Comment:
Checked again, the extra space is due to moving the newline to header_prefix. It was counted as an extra character during formatting. So this is correct.